### ISSUE TITLE
stacked nav help

### ISSUE THREAD

**Issue by rachelhisko** [rachelhisko/stacked-nav#1]
I need help setting up html file and than I guess the css. 
@thomasjbradley

**Comment by thomasjbradley** [comment]
You're definitely on the right track so far.

There's a couple changes I'd make so far:
1. Move the `main.css` into a `css` folder
2. Change the HTML for the navigation so that each item is an `<li>`—and since they are navigation, they should also probably be `<a>` tags.

Here's some references for making navigation:
https://learn-the-web.algonquindesign.ca/topics/navigation/

**Comment by rachelhisko** [comment]
I always have problems with putting the main.css into the css folder?
I always think I do it write but i guess it never works?
How do I put it in the css folder?

**Comment by thomasjbradley** [comment]
1. In Finder, go to your `responsive-nav` folder.
2. Right click in `responsive-nav` and press “New Folder”, name it `css`.
3. Drag the `main.css` file into the new `css` folder.

**Comment by rachelhisko** [comment]
okay thanks!
I think I got the nav part right.
Although now I am trying to figure out how to do the tittle?
Also the screenshot is center I do that in the css file right?

**Comment by thomasjbradley** [comment]
> I think I got the nav part right.

Yep, looks good to me.

> Although now I am trying to figure out how to do the tittle?

What about using an `<h1>`, since it's the most important piece of content on the page?

> Also the screenshot is center I do that in the css file right?

Yeppers, you do the centering in your CSS file.

**Comment by rachelhisko** [comment]
Am i know done with the html file?

**Comment by thomasjbradley** [comment]
The closing `</body>` and closing `</html`> tags are missing at the bottom, but otherwise it should be okay.

One improvement would be: should there be a `<header>` tag?

**Comment by rachelhisko** [comment]
I am not sure? do I have one?

and how do I make these two come together. Like the colors touch?
![screen shot 2016-02-19 at 2 54 49 pm](https://cloud.githubusercontent.com/assets/16649625/13187396/c5d4f78c-d718-11e5-9285-92ce0b81c2d8.png)

**Comment by thomasjbradley** [comment]
> and how do I make these two come together. Like the colors touch?

Have you tried using the developer tools to inspect and see what is causing those spaces?

**Comment by rachelhisko** [comment]
well i know for one I have to create a box around the whole html but I can't?
![screen shot 2016-02-19 at 3 04 22 pm](https://cloud.githubusercontent.com/assets/16649625/13187598/171c2786-d71a-11e5-986e-b9dac14ce943.png)

**Comment by thomasjbradley** [comment]
In the other screenshot, just above, what element was the darker brown colour on in the CSS? Why did you move the colour onto each link?

**Comment by rachelhisko** [comment]
I was trying to make it into a box, not have it look like the title. because they are both done wrong.

**Comment by thomasjbradley** [comment]
What do you mean they are both “done wrong”? The first screenshot you sent looks perfectly inline with what I expect to be happening. It just needed a few things removed like the bullets and extra space; and a few things added.

**Comment by rachelhisko** [comment]
OH I thought it had to be in a box because there is no white surrounding it?

**Comment by thomasjbradley** [comment]
Nope, you were on the right track.

The white is caused by some of the browsers default CSS. Using the developer tools we can figure out where the white is coming from and overwrite it in our CSS.

**Comment by rachelhisko** [comment]
okay I put it back. Mark-bot says what you said about the header?
![screen shot 2016-02-19 at 3 25 00 pm](https://cloud.githubusercontent.com/assets/16649625/13188035/f60b082a-d71c-11e5-8111-7da289bde071.png)

**Comment by thomasjbradley** [comment]
Yep, Markbot is expecting to see the `<header>` element in the HTML.

**Comment by rachelhisko** [comment]
sorry I don't understand where?

**Comment by thomasjbradley** [comment]
Do you remember what the `<header>` element is used for?

**Comment by rachelhisko** [comment]
no sorry I don't, is there a video about it ?

**Comment by thomasjbradley** [comment]
There sure is!

Here's the written tutorial:
https://learn-the-web.algonquindesign.ca/topics/html-semantics/#document-elements

Here's the video:
https://www.youtube.com/watch?list=PLWjCJDeWfDdc0Sp_DinOWnodw3KnWCwc1&v=QnRBrDgeOuw

**Comment by rachelhisko** [comment]
Thank you I understand now!
Markbot says something about how my colors I got are not a background value.
![screen shot 2016-02-19 at 3 40 00 pm](https://cloud.githubusercontent.com/assets/16649625/13188360/10cd9e28-d71f-11e5-99a5-62620d7d3f90.png)

**Comment by thomasjbradley** [comment]
They're missing the `#` at the start of the colour value.

**Comment by rachelhisko** [comment]
Thank you very much for your time!
